src/sentry/models/release.py

```diff
@@ -56,6 +56,9 @@ class ReleaseProject(Model):
     release = FlexibleForeignKey("sentry.Release")
     new_groups = BoundedPositiveIntegerField(null=True, default=0)
 
+    adopted = models.DateTimeField(null=True, blank=True, db_index=True)
+    unadopted = models.DateTimeField(null=True, blank=True, db_index=True)
```

Comment:
I don't think these indexes will work for the query patterns you're likely to make. I think based on reqs, our likely use case is:

"Give me all (un)adopted releases for a project". and/or "Sort all releases for a project by their (un)adoption date"

Reply:
Also slightly tricky since we'll want to query across many projects sometimes, probably not much we can do about that though unless we want to add org id here